spec/uvobot_spec.rb

```diff
@@ -1,13 +1,11 @@
-require 'spec_helper'
 require './lib/uvobot'
-require './lib/slack_notifier'
 require './lib/uvo_scraper'
 
 RSpec.describe Uvobot do
   describe '.run' do
     let(:notifier) { double }
     let(:scraper) { double }
-    let(:bot) { Uvobot.new(notifier, scraper) }
+    let(:bot) { Uvobot.new([notifier], scraper) }
```

Comment:
Zvyknem davat notifierov ako posledny argument. Kedze niekedy sa tam dava aj `DummyNotifier` a potom to vyzera divne.